TimePad/BaseTabBarController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ViewController.swift
+//  TimePad
+//
+//  Created by Emre Alpago on 10.09.2022.
+//
+
+import UIKit
+
+enum TabbarItemType {
+    case home, add, pie
+    
+    func image() -> UIImage {
+        switch self {
+        case .home:
+            return UIImage(named: "time")!
+        case .add:
+            return UIImage(named: "add")!
+        case .pie:
+            return UIImage(named: "pie")!
+        }
+    }
+    
+    func selectedImage() -> UIImage {
+        switch self {
+        case .home:
+            return UIImage(named: "time_Selected")!
+        case .add:
+            return UIImage(named: "add_Selected")!
+        case .pie:
+            return UIImage(named: "pie_Selected")!
+        }
+    }
+}
+
+
+class BaseTabBarController: UITabBarController {
+    
+    var items: [TabbarItemType] {
+        get {
+            return [.home, .add ,.pie]
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        setupTabBar()
+        UITabBar.appearance().backgroundColor = UIColor.white
```

Comment:
background color'ın tipi UIColor o yüzden direkt olarak .white diyebilirsin
```suggestion
        UITabBar.appearance().backgroundColor = .white
```